tests/components/nut/test_config_flow.py

```diff
@@ -30,27 +58,82 @@ def _get_mock_pynutclient(list_vars=None):
     ) as mock_setup, patch(
         "homeassistant.components.nut.async_setup_entry", return_value=True,
     ) as mock_setup_entry:
+        result3 = await hass.config_entries.flow.async_configure(
+            result2["flow_id"], {"resources": ["battery.voltage"]},
+        )
+
+    assert result3["type"] == "create_entry"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5329694)